### PR TITLE
[DOCFIX] Update dora docs

### PR DESCRIPTION
**Pull request by LuQQiu**
Update dora docs to be able to show on Alluxio docs website

### CODE REVIEW

**Comment by Xenorith** [comment]
@LuQQiu in a separate PR, we should generate the property keys and metrics

File: docs/en/Overview.md [hunk 1]
```diff
@@ -34,87 +34,107 @@ with the largest deployment exceeding 1,500 nodes.
...
+Each worker node can support tens of millions of files, making it easy to handle increasing data volumes and growing user bases.
 
-To quickly get Alluxio up and running, take a look at our
-[Getting Started]({{ '/en/overview/Getting-Started.html' | relativize_url }}) page,
-which explains how to deploy Alluxio and run examples in a local environment.
+### Paging Data Store
 
-Also try our getting started tutorial for Presto & Alluxio via:
+DORA uses a paging store model as its cache storage, offering finer-grained caching for small to medium-sized read requests on large files.
+DORA's fine-grained caching has resulted in up to 150x read amplification and improved unstructured file position read by up to 9x.
```
Comment (Xenorith) [inline docs/en/Overview.md:79]: performance comparisons should explain the reference. 150x from what? prefer to remove them as they don't offer the reader any useful info. should explain what the concept is and why it is a good choice, not spit out baseless comparisons

Comment (LuQQiu) [reply docs/en/Overview.md:79]: lol those are copied from blog....

File: docs/en/Overview.md [hunk 2]
```diff
@@ -34,87 +34,117 @@ with the largest deployment exceeding 1,500 nodes.
...
+DORA uses a page store module as its cache storage, offering finer-grained caching for small to medium-sized read requests on large files.
+This page store has been battle-tested in applications like Presto in Meta, Uber, and TikTok, proving its reliability.
+DORA's fine-grained caching has resulted in solving up to 150X read amplification issues and improving unstructured file position read up to 9X.
+Additionally, it has improved structured file position read by 2 to 15X.
 
-<p align="center">
-<a href="https://www.alluxio.io/alluxio-presto-sandbox-docker/">
- <img src="https://www.alluxio.io/app/uploads/2019/07/laptop-docker.png" width="250" alt="Laptop with Docker"/></a>
-</p>
+![Dora read approaches]({{ '/img/dora_read_approaches.png' | relativize_url }})
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply docs/en/Overview.md:82]: position read is for structured file, e.g. read a column from the table
sequential read is for unstructured file, e.g. read a whole image, video, etc

**Review by Xenorith — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**